README.md

```diff
@@ -1,786 +1,104 @@
+1. People that find your software can easily cite it using the correct metadata from `CITATION.cff`!
+2. If your repository is hosted on GitHub, [they will link to it](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-citation-files) in a special "Cite this repository" widget on the repository landing page. Also, the citation information in `CITATION.cff` will be used when you publish your software on Zenodo via the [GitHub-Zenodo bridge](https://guides.github.com/activities/citable-code/).
+3. The citation information in `CITATION.cff` can be used by the open source reference manager [Zotero](https://github.com/zotero/zotero), when people import the repository.
 
-For a work that is complete and has been published, leave `status` unset.
+## Validation :heavy_check_mark:
 
-## License strings
+## Tools to work with `CITATION.cff` files :wrench:
 
-License strings must conform with the [SPDX Licenses
-list](https://spdx.org/licenses/), i.e., a license must be specified via the
-short identifier from the list. If a license is not included in the SPDX
-Licenses list, the `license-url` should be provided as a fallback.
+## Maintainers :nerd_face:
 
-Example:
+The Citation File Format schema is maintained by
 
-```yaml
-references:
-  - type: software
-    authors:
-      - ...
-    title: My Research Tool
-    license: Apache-2.0
-  - type: software
-    authors:
-      - ...
-    title: Obscure Research Tool
-    license-url: http://r3s34archs0ft.com/eula
-```
+- [Stephan Druskat](https://sdruskat.net) ([@sdruskat](https://github.com/sdruskat/))
+- Jurriaan Spaaks ([@jspaaks](https://github.com/jspaaks/))
 
-## Language strings
+## Contributing :handshake:
 
-Natural languages as a value for the key `languages` are specified via their
-respective 3-character [ISO 639-3](https://en.wikipedia.org/wiki/ISO_639-3) code. A list
-of ISO 639-3 codes in maintained at
-[Wikipedia:List of ISO 639-3 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-3_codes).
-Alternatively, a language's 2-character
-[ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) code may be used. A list
-of ISO 639-1 codes is maintained at
-[Wikipedia:List of ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).
+**The _Citation File Format_ is a collaborative project** and we welcome suggestions and contributions. We hope one of the invitations below works for you, but if not, please let us know!
 
-Example for a work in both English and Daakaka:
+:running: **I'm busy, I only have 1 minute**
+- Tell a friend about the *Citation File Format*!
 
-```yaml
-references:
-  - type: book
-    ...
-    languages:
-      - en
-      - bpa
-```
+:hourglass_flowing_sand: **I've got 5 minutes - tell me what I should do**
+- Create a `CITATION.cff` file for your repository.
+- Suggest ideas for how you would like to use the *Citation File Format*, or for an improvement to the format or its tooling.
```

Comment:
This is a bit risky, because it will likely take more than 5 minutes, and then we will have betrayed the trust that people put in this project the very first time we met